libavfilter/f_ebur128.c

```diff
@@ -137,6 +137,10 @@ typedef struct EBUR128Context {
     int target;                     ///< target level in LUFS used to set relative zero LU in visualization
     int gauge_type;                 ///< whether gauge shows momentary or short
     int scale;                      ///< display scale type of statistics
+    int gate_measurement;           ///< whether or not to use the gated measurement
+    double *ch_raw_integrated_power; ///< each channels's power
```

Comment:
Can we move the 3 variables below gate measurement out of the `/* misc */` section into the `/* I and LRA Specific */` section on line 127.

Additionally I think a more accurate name from `ch_raw_integrated_power` is `ch_ungated_filtered_sample`

Reply:
Additionally change the comment to `///< the unfiltered (BS.1770-1) per channel sample`

Reply:
Sorry, I think I was confusing the differences between BS.1770-1/2 with BS.1770-2/3

I think it should be `ch_ungated_sample` and `///< the ungated (BS.1770-1) per channel sample`